packages/storybook/src/web-components/button/button.mdx

```diff
@@ -37,7 +37,7 @@ niet bestaat in onze tokens dan zal je een paarse styling zien, als visuele waar
 <Description of={ButtonStories.Tertiary} />
 <Canvas of={ButtonStories.Tertiary} />
 
-## States
+<h2 id="states_examples">States</h2>
```

Comment:
Y tho?

Reply:
De Readme heeft al een `##States`, die steelt de anchor in TOC